src/convertToObject.js

```diff
@@ -3,16 +3,54 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example in
+ * [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties (see an
+ * exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const splitStrings = sourceString.split(';');
+
+  const splitLongStrings = splitStrings.filter(x => x.length > 4);
+
+  const stringsSplitedColon = [];
+
+  splitLongStrings.forEach(x => {
+    const arrayCopy = x.split(':');
+
+    stringsSplitedColon.push(arrayCopy[0]);
+    stringsSplitedColon.push(arrayCopy[1]);
+  });
+
+  const stringsWithoutSpaces = stringsSplitedColon.map(x => {
```

Comment:
2. Then this whole thing could be replace by [trim](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trim):
```js
const trimmedPropertiesAndValuesArr = propertiesAndKeysArr.map(x => x.trim());
```

Reply:
This will work and make the function return exact same result